File: Maximum Value of an Ordered Triplet I.js
function maximumTripletValue(nums){
    //initialize answer
    let ans = 0;
    //3 nesteed for loops to check all triplets
    for (let i = 0; i < nums.length; i++){
        for (let j = i + 1; j < nums.length; j++){
            for (let k = j + 1; k < nums.length; k++){
                if ((nums[i] - nums[j]) * nums[k] > ans){
                    ans = (nums[i] - nums[j]) * nums[k];
                }
            }
        }
    }
    return ans;
}

File: Maximum Value of an Ordered Triplet I.go
package main

func maximumTripletValue(nums []int) int64 {
	ans := int64(0)
	for i := 0; i < len(nums); i++ {
		for j := i + 1; j < len(nums); j++ {
			for k := j + 1; k < len(nums); k++ {
				current := int64(nums[i]-nums[j]) * int64(nums[k])
				if current > ans {
					ans = current // Update ans if current is greater
				}

			}
		}
	}
	return ans
}
